packages/css-reset/README.md

```diff
@@ -1,3 +1,20 @@
-# TDS Core: CSS Reset
+```css noeditor
+// in jsx
+import '@tds/core-reset/dist/index.css'
 
-Global CSS rules to apply consistent base styles, including a CSS reset, font faces, and root styles.
+// in Sass
+import "~@tds/core-reset/dist/index"
```

Comment:
I know I gave the example..... sorry, just more miscommunication. :(

I think its better NOT to include the Sass way here. We don't want people to do that, so better just to leave it out altogether.

Reply:
I removed the sass example and also added a note to use in combination with [ignore-styles](https://www.npmjs.com/package/ignore-styles) npm package